ci/build_docs.sh

```diff
@@ -6,12 +6,14 @@ set -euo pipefail
 rapids-logger "Create test conda environment"
 . /opt/conda/etc/profile.d/conda.sh
 
+ENV_YAML_DIR="$(mktemp -d)"
```

Comment:
Not strictly necessary since these scripts almost always run in a container, but cleaning up temp directories isn't a bad idea.

```suggestion
ENV_YAML_DIR="$(mktemp -d)"
trap 'rm -rf -- "$ENV_YAML_DIR"' EXIT
```

Reply:
Aren't temp directories cleared on boot automatically? I'd vote to omit this extra line to keep things as succinct as possible.